tests/object_factor_base_test.php

```diff
@@ -25,7 +25,7 @@
  * @copyright   2023 Catalyst IT
  * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
-class manager_test extends tool_mfa_testcase {
+class base_factor_test extends tool_mfa_testcase {
```

Comment:
should it match file name or vice versa? 
So either class should be object_factor_base_test or file should be base_factor_test.php

Reply:
Fixed